ci-operator/step-registry/windows/conf/operator/windows-conf-operator-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
```

Comment:
Can't you somehow leverage the [operatorhub](https://github.com/openshift/release/tree/master/ci-operator/step-registry/operatorhub) steps?